src/components/apps/editor/index.js

```diff
@@ -223,6 +227,11 @@ const AppEditor = (props) => {
                 const app = formatSubmission(values);
 
                 const onSuccess = (app) => {
+                    if (!values.id) {
+                        // A new app was saved, so update address to new URL
+                        router.replace(getAppEditPath(app.system_id, app.id));
+                    }
+
```

Comment:
Not sure if this is possible but putting it out here anyway. When I save the app from a step 2 , for the first time, after saving the app, I got to step 1. It would be nice to remember the step and return the users to the same step. May be a query param to remember the step number ?